### DISCUSSION TITLE
Why doesnt networking cables and monitors not work?

### DISCUSSION THREAD

**Discussion by giftedl** [cc-tweaked/CC-Tweaked#1389] [Q&A] [answered]
![2023-03-25](https://user-images.githubusercontent.com/52332868/227779622-118cc245-c8e5-4f3b-b104-127dc03919ba.png)
![2023-03-26 (1)](https://user-images.githubusercontent.com/52332868/227779624-0002c04e-3f1c-4285-8bcd-44f435154908.png)
![2023-03-26 (2)](https://user-images.githubusercontent.com/52332868/227779627-4fd8704d-7c6e-4352-8f4d-ea0995f2050b.png)

**Comment by Wojbie** [comment]
Looks like something odd is happening here. What mods do you have installed? Is this clean CC:T instance or with some other mods. Which exact CC:T version and on which modloader? In both cases can you run program `peripherals` and screenshot what it says?

**Comment by giftedl** [comment]
Optifine and cc tweaks.

Forge is the modloader.

**Reply by giftedl** [reply]
and i did run `peripherals`

![2023-03-26 (3)](https://user-images.githubusercontent.com/52332868/227780476-062cdd7e-bf6d-4fd1-97d8-e9d59a699c29.png)

**Comment by giftedl** [comment, marked as answer]
I just had to remove optifine.... thanks for the help :)
